Makefile

```diff
@@ -347,16 +347,16 @@ build-nogtfs-%: otp.jar
 
 
 clean-all:
-	find . -name osm.pbf -printf '%p\n' -exec rm {} \;
-	find . -name gtfs.zip -printf '%p\n' -exec rm {} \;
-	find . -name graph.obj -printf '%p\n' -exec rm {} \;
-	find . -name streetGraph.obj -printf '%p\n' -exec rm {} \;
+	find . -name osm.pbf -print -exec rm {} \;
+	find . -name gtfs.zip -print -exec rm {} \;
+	find . -name graph.obj -print -exec rm {} \;
+	find . -name streetGraph.obj -print -exec rm {} \;
 	rm -f otp.jar
 
 clean-%:
-	find $* -name *gtfs.zip -printf '%p\n' -exec rm {} \;
```

Comment:
Can you please restore the `$*`?

That is useful for only cleaning a single location, not all of them.

Reply:
`$* doesn't work on my Mac
<img width="509" alt="image" src="https://user-images.githubusercontent.com/62163307/231819950-0b7e23b9-f03f-4209-934a-88ad8bc7abbd.png">

Reply:
That is a make variable. It will be replace with "portland" for example.

Reply:
You're right, made the fix!